build/jobs/package.yml

```diff
@@ -92,3 +83,5 @@ steps:
       pathToPublish: '$(build.binariesdirectory)/IntegrationTests'
       artifactName: 'IntegrationTests'
       artifactType: 'container'
+      parallel: true
+      parallelCount: 32
```

Comment:
This doesn't look like it is making this step faster. The time seems to be ~5 minutes with or without it.

Reply:
Worth a shot, it didn't, I'll revert